src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java

```diff
@@ -52,10 +52,11 @@ public void testGetMicrosoftTimeZoneNameBad() {
     } catch (final IllegalArgumentException ignored) {}
```

Comment:
while looking at this I was wondering what we want to do here? `Assert.fail(TimeZoneUtils.getMicrosoftTimeZoneName(null));` I think this test should handle a `@Test(expected = IllegalArgumentException.class` for the null vlaue